pkg/remote/remote.go

```diff
@@ -513,7 +513,9 @@ func createDockerignoreFileWithFilesystem(cwd, tmpDir string, rules []string, us
 		dockerignoreContent = append(dockerignoreContent, []byte(rule)...)
 		dockerignoreContent = append(dockerignoreContent, []byte("\n")...)
 	}
-
+	if len(dockerignoreContent) == 0 {
+		dockerignoreContent = []byte("# Okteto docker ignore\n")
+	}
```

Comment:
Could we add a test case in `TestCreateDockerignoreFileWithFilesystem` for this scenario?